src/components/popover/popover.css

```diff
@@ -0,0 +1,32 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography-v2.css';
+
+.close-button {
+  composes: button from '../icon-button/icon-button.css';
+  height: 12px;
+  margin-left: var(--spacing-large);
+  padding: var(--spacing-small);
+  padding-top: 5px;
```

Comment:
I'm not sure we need the pixel correction here. But I only briefly looked at it when toggling this style.

Reply:
It's pretty subtle, but it definitely isn't center-aligned with the text without it. 😭